vNext/extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts

```diff
@@ -114,6 +116,11 @@ export default class PropertiesPlugin implements ITelemetryPlugin, IPropertiesPl
         this.context.applyOperatingSystemContxt(event);
         this.context.applyWebContext(event);
 
+        if (!event.data) {
+            event.data = {};
+        }
+        this.context.applyCustomProperties(event);
```

Comment:
This is already done in the applicationinsights-analytics SKU. Why is this needed in properties plugin?